karma.config.js

```diff
@@ -0,0 +1,67 @@
+/*jslint node: true */
+'use strict';
+
+/**
+ * Common Karma configuration shared between local / CI testing.
+ * @name getConfig
+ */
+function getConfig(config) {
+
+  // This file is spawned so we'll need to read the args again
+  const minimist = require('minimist');
+  const argv = minimist(process.argv.slice(2));
+  const path = require('path');
+
+  const testWebpackConfig = require('./node_modules/@blackbaud/skyux-builder/config/webpack/test.webpack.config');
+  const remapIstanbul = require('remap-istanbul');
+  const skyPagesConfig = require('./node_modules/@blackbaud/skyux-builder/config/sky-pages/sky-pages.config').getSkyPagesConfig(argv.command);
+
+  config.set({
+    basePath: '',
+    frameworks: ['jasmine'],
+    browsers: [
+      'Chrome'
+    ],
+    exclude: [],
+    files: [
+      {
+        pattern: './node_modules/@blackbaud/skyux-builder/utils/spec-bundle.js',
+        watched: false
+      },
+      {
+        pattern: './node_modules/@blackbaud/skyux-builder/utils/spec-styles.js',
+        watched: false
+      }
+    ],
+    preprocessors: {
+      './node_modules/@blackbaud/skyux-builder/utils/spec-styles.js': ['webpack'],
+      './node_modules/@blackbaud/skyux-builder/utils/spec-bundle.js': ['coverage', 'webpack', 'sourcemap']
+    },
+    webpack: testWebpackConfig.getWebpackConfig(skyPagesConfig),
+    coverageReporter: {
+      dir: path.join(process.cwd(), 'coverage'),
+      reporters: [
+        // these reporters are incompatible with `istanbul-instrumenter-loader v1.0.0`
+        // use `v0.2.0` instead if you need to use these reporters
+        { type: 'json' },
+        { type: 'html' }
+      ],
+      _onWriteReport: function (collector) {
+        return remapIstanbul.remap(collector.getFinalCoverage());
+      }
+    },
+    webpackServer: {
+      noInfo: true,
+      stats: 'minimal'
+    },
+    reporters: ['mocha', 'coverage'],
+    port: 9876,
+    colors: true,
+    logLevel: config.LOG_INFO,
+    autoWatch: false,
+    singleRun: true,
+    browserNoActivityTimeout: 100000
```

Comment:
@blackbaud-brandonhare This line is what made it work. Apparently, the wrapper tests are taking quite a while and Chrome timed out. IMO we have two options: 1) Double-check the tests to make sure the code isn't caught in a time warp, or 2) submit a PR to Builder to add the `browserNoActivityTimeout` to the test config.